### PR TITLE
[EOC-107] Additional info on list tiles

### PR DESCRIPTION
**Pull request by alemikolo**
list_tiles_details

### CODE REVIEW

**Comment by adamklepacz** [comment]
https://ux.stackexchange.com/questions/97032/what-is-the-best-static-icon-to-represent-work-in-progress

Consider using icons for Done and Unhadled.
https://fontawesome.com/icons/clock?style=regular
https://fontawesome.com/icons/check?style=solid

File: server/common/utils/index.js [hunk 1]
```diff
@@ -18,13 +18,37 @@ const isValidMongoId = id => ObjectId.isValid(id);
 
 const isUserFavourite = (favIds, userId) => favIds.indexOf(userId) > -1;
 
+const responseWithList = (list, userId) => {
+  const { _id, cohortId, description, favIds, isPrivate, items, name } = list;
+  const doneItemsCount = items.filter(item => item.isOrdered).length;
+
```
Comment (adamklepacz) [inline server/common/utils/index.js]: `const unhandledItemsCount = items.lenght - doneItemsCount`

File: server/common/utils/index.js [hunk 2]
```diff
@@ -18,13 +18,37 @@ const isValidMongoId = id => ObjectId.isValid(id);
...
+  const listToSend = {
+    _id,
+    description,
+    doneItemsCount,
+    isFavourite: isUserFavourite(favIds, userId),
+    isPrivate,
+    name,
+    unhandledItemsCount: items.length - doneItemsCount
```
Comment (adamklepacz) [inline server/common/utils/index.js]: just `unhandledItemsCount`

File: server/common/utils/index.js [hunk 3]
```diff
@@ -18,13 +18,37 @@ const isValidMongoId = id => ObjectId.isValid(id);
...
-    const { favIds, ...rest } = list._doc;
+    const { favIds, items, ...rest } = list._doc;
+    const doneItemsCount = items.filter(item => item.isOrdered).length;
 
     return {
       ...rest,
-      isFavourite: isUserFavourite(favIds, userId)
+      doneItemsCount,
+      isFavourite: isUserFavourite(favIds, userId),
+      unhandledItemsCount: items.length - doneItemsCount
```
Comment (adamklepacz) [inline server/common/utils/index.js]: same as in the above comments

**Comment by alemikolo** [comment]
https://ux.stackexchange.com/questions/97032/what-is-the-best-static-icon-to-represent-work-in-progress

Consider using icons for Done and Unhadled.
https://fontawesome.com/icons/clock?style=regular
https://fontawesome.com/icons/check?style=solid

Let's wait what Łukasz says. I made a note that in the tiles view, do not use icons. In addition, icons or initials are not explicit information especially for visually impaired people.

**Pull request merged**